lib/OpenQA/WebAPI/Controller/Admin/Machine.pm

```diff
@@ -19,9 +19,7 @@ use strict;
 use parent 'OpenQA::WebAPI::Controller::Admin::Table';
 
 sub index {
-    my $self = shift;
-
-    $self->SUPER::admintable('MachineSettings', 'machine');
+    shift->SUPER::admintable('machine');
```

Comment:
Maybe we can remove this entire method (also in `Product.pm` and `TestSuite.pm`).